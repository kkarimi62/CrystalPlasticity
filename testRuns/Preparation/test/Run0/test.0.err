  File "pyScripy.py", line 1
    import imp; gn=imp.load_source('generate.name',/home/kamran.karimi1/Project/git/CrystalPlasticity/py/generate.py);  							   						gn.Generate( 1000, 5,(0.0,1.0), (0.0, 1.0), (0.0, 1.0), 						 							title = 'data.txt', 			 										ratio1 = 0.05, ratio2 = 0.26, ratio3 = 0.02, ratio4 = 0.4, ratio5 = 0.27 )
                                                   ^
SyntaxError: invalid syntax
