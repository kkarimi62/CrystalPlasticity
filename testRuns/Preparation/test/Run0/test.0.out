LAMMPS (29 Oct 2020)
Reading data file ...
  orthogonal box = (0.0000000 0.0000000 0.0000000) to (1.0000000 1.0000000 1.0000000)
  1 by 1 by 1 MPI processor grid
  reading atoms ...
  1000 atoms
  read_data CPU = 0.003 seconds
ERROR: Incorrect args for pair coefficients (../pair_meamc.cpp:212)
Last command:   pair_coeff      * *  ${pairCoeffArgs} #library_CoNiCrFeMn.meam Co Ni Cr Fe Mn parameters.meam Co Ni Cr Fe Mn
